app/views/triage/whatDoYouWantToDo.scala.html

```diff
@@ -15,37 +15,53 @@
  *@
 
 @import config.FrontendAppConfig
-@import uk.gov.hmrc.play.views.html._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.{FormWithCSRF => PlayFrontendFormWithCSRF}
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
 @import controllers.triage.routes._
 @import models.{Mode}
 @import models.triage.WhatDoYouWantToDo
 @import utils.ViewHelper.titleWithErrors
 
 @this(
-    main_template: main_template,
-    formHelper: FormWithCSRF
+layout: templates.Layout,
+govukRadios : GovukRadios,
+formHelper: PlayFrontendFormWithCSRF,
+govukErrorSummary: GovukErrorSummary,
+button: components.govuk_button
 )
 
 @(form: Form[_], role: String)(implicit request: Request[_], messages: Messages)
 
-@main_template(
-    title = titleWithErrors(form, messages("messages__whatDoYouWantToDo__title"))) {
+@layout(
+pageTitle = titleWithErrors(form, messages("messages__whatDoYouWantToDo__title"))) {
 
-    @formHelper(action = controllers.triage.routes.WhatDoYouWantToDoController.onSubmit(role), 'autoComplete -> "off") {
+@formHelper(action = controllers.triage.routes.WhatDoYouWantToDoController.onSubmit(role), 'autoComplete -> "off") {
+
+@if(form.errors.nonEmpty) {
+@govukErrorSummary(ErrorSummary().withFormErrorsAsText(form))
+}
 
-        @components.back_link()
 
-        @components.error_summary(form.errors)
+@govukRadios(
+Radios(
+fieldset = Some(Fieldset(
+legend = Some(Legend(
+content = Text(messages("messages__whatDoYouWantToDo__title")),
```

Comment:
Can this be formatted, ie indented correctly? I won't put any further formatting comments in. Maybe just go through and check each of them is formatted properly?